frameworks/OCaml/opium/src/dune-project

```diff
@@ -0,0 +1,47 @@
+(lang dune 2.7)
+
+(name opi)
+
+(generate_opam_files true)
+
+(source
+ (github TechEmpower/FrameworkBenchmarks))
+
+(license MIT)
+
+(authors "Robin Bj\195\182rklin" "mud")
```

Comment:
I'm not sure my name deserves to be on the opium implementation but thanks anyway :) 

PS. What happened to the letter `ö` it seems to have been weirdly escaped? "Björklin" or "Bjorklin" would also be perfectly fine.

Reply:
hmm that's weird, I'm using `sheerun/vim-polyglot` and `ocaml-lsp-server`, one of these autoformats `dune-project` files with these escapes

re the authors, I copied most of the configs from your implementation, so it's well deserved :)